src/datastore/pg-store.ts

```diff
@@ -1468,6 +1483,15 @@ export class PgStore extends BasePgStore {
       const contractIdFilterSql = contractId
         ? sql`AND contract_call_contract_id = ${contractId}`
         : sql``;
+
+      const pgTrgmInstalled = await this.isPgTrgmInstalled(sql);
```

Comment:
Can this be inlined to avoid another round-trip query to postgres? E.g. something like
```ts
const searchTermFilterSql = searchTerm
  ? sql`
      AND (
        CASE
          WHEN EXISTS (
            SELECT 1
            FROM pg_extension
            WHERE extname = 'pg_trgm'
          )
          THEN similarity(contract_call_function_name, ${searchTerm}) > ${TRGM_SIMILARITY_THRESHOLD}
          ELSE contract_call_function_name ILIKE '%' || ${searchTerm} || '%'
        END
      )
    `
  : sql``;
```